picard/webservice/__init__.py

```diff
@@ -134,6 +136,8 @@ def __init__(
             important: Indicates that this is an important request.
             request_mimetype: Set the Content-Type header.
             url: URL passed as a string or as a QUrl to use for this request
+            queryargs: Encoded query arguments, a dictionary mapping field names to values
+            unencoded_queryargs: Unencoded query arguments, a dictionary mapping field names to values
```

Comment:
I think we should not have both, only one parameter accepting unencoded query args. Or do we need that queryargs argument to be encoded for some use case?

Reply:
I think keeping both doesn't hurt for now, but getting rid of `queryargs` is my goal.
We'll do another round of cleanup.
The PR is big enough ;)

There's also `MUSICBRAINZ_SERVERS` to look into (host, port -> URL)